Assets/StarterAssets/FirstPersonController/Scripts/FirstPersonController.cs

```diff
@@ -74,11 +73,13 @@ public class FirstPersonController : MonoBehaviour
 		private float _terminalVelocity = 53.0f;
         private float currentStamina;
         //private int currentJumps = 0;
-        private bool extraJump = false;
+        private bool extraJump = true;
```

Comment:
Allow extraJump from start (Probably not necessary since it will be set to "true" when grounded)